Constraints/DataLoader.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DataLoader.swift
+//  Constraints
+//
+//  Created by Agustin Godoy Cosser on 16/08/2022.
+//
+
+import Foundation
+
+public class DataLoader {
+    
+   func loadMessages() -> [MessageData] {
+        
+        if let fileLocation = Bundle.main.url(forResource: "messages", withExtension: "json") {
```

Comment:
just a comment.. the resource should be as a parameter of the function

Reply:
https://github.com/agustingodoyc/Constraints/pull/3/commits/cf7bd49e01a4de1765fca2b2d9859ea67728b93f

Reply:
https://github.com/agustingodoyc/Constraints/pull/3/commits/4d492815c7dee4cf4d6ed2a22942dde057df03da